webapp/channels/src/components/admin_console/team_channel_settings/group/group_list.tsx

```diff
@@ -40,18 +40,18 @@ const Header = () => {
     );
 };
 
-interface Props {
+type Props = {
```

Comment:
this can be just Props = OwnProps & PropsFromRedux

Reply:
Hi @M-ZubairAhmed, I updated the line as you suggested. I also included additional props in the index.ts file and updated the group_list_test file as well. However, I keep getting the following error (even after removing the old snapshot file and adding the --updateSnapshot flag while running `npm test` on the test file): `Could not find "store" in the context of "Connect(GroupList)". Either wrap the root component in a <Provider>, or pass a custom React context provider to <Provider> and the corresponding React context consumer to Connect(GroupList) in connect options.`